presto-docs/src/main/sphinx/presto_cpp/limitations.rst

```diff
@@ -0,0 +1,44 @@
+======================
+Presto C++ Limitations
+======================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+General Limitations
+===================
+
+The C++ evaluation engine has a number of limitations:
+
+* Not all built-in functions are implemented in C++. Attempting to use unimplemented functions results in a query failure. For supported functions, see `Function Coverage <https://facebookincubator.github.io/velox/functions/presto/coverage.html>`_.
+
+* Not all built-in types are implemented in C++. Attempting to use unimplemented types will result in a query failure. 
```

Comment:
@steveburnett : Apologize for the delay. I feel we should add more detail here about the supported and unsupported types here. 

The specifics are :
All basic/structured types (from https://prestodb.io/docs/0.286/language/types.html) but [CHAR, TIME, TIME WITH TIMEZONE] are supported. These are subsumed by VARCHAR, TIMESTAMP and TIMESTAMP WITH TIMEZONE. 

Caveat : Prestissimo only supports unlimited length Varchar. It does not honor the lengths in varchar[n].

Types IPADDRESS, IPPREFIX, UUID, kHYPERLOGLOG, P4HYPERLOGLOG, QDIGEST, TDIGEST are not supported

Reply:
Opened #22772 to add this content to the docs.